static/js/components/email/EmailCompositionDialog.js

```diff
@@ -78,6 +79,9 @@ export default class EmailCompositionDialog extends React.Component {
   constructor (props: EmailDialogProps) {
     super(props);
     this.state = editorStateFromProps(props);
+    if (R.isNil(this.state.editorState)) {
```

Comment:
She needs to take care of the case where the first action the user does is to click one of the variable buttons, before `Editor` triggers `onEditorStateChange`